tests/components/nextdns/__init__.py

```diff
@@ -157,6 +151,20 @@ async def init_integration(hass: HomeAssistant) -> MockConfigEntry:
             return_value=CONNECTION_STATUS,
         ),
     ):
+        yield
+
+
+async def init_integration(hass: HomeAssistant) -> MockConfigEntry:
+    """Set up the NextDNS integration in Home Assistant."""
+    entry = MockConfigEntry(
+        domain=DOMAIN,
+        title="Fake Profile",
+        unique_id="xyz12",
+        data={CONF_API_KEY: "fake_api_key", CONF_PROFILE_ID: "xyz12"},
+        entry_id="d9aa37407ddac7b964a99e86312288d6",
+    )
+
+    with mock_nextdns():
         entry.add_to_hass(hass)
```

Comment:
We can move this line up out of the context manager.

Reply:
c003d0a767f40093850b95a8b1ca0f81840a7aa9